src/gatsby/src/components/search/cityNameSearch.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import { Typeahead, ClearButton, Menu, MenuItem } from "react-bootstrap-typeahead"
+import { Form } from "react-bootstrap"
+import PermissionToast from "./permissionToast"
+import NearMeIcon from "@mui/icons-material/NearMe"
+import "react-bootstrap-typeahead/css/Typeahead.css"
+
+const HighlightText = ({ city, input }) => {
+  const words = city.split(" ")
+  return (
+    words.map((word, index) => {
+      if (word.toLowerCase() === input) {
+        return <span key={index}> {word} </span>
+      } else {
+        return <b key={index}> {word} </b>
+      }
+    })
+  )
+}
+
+const CityNameSearch = ({
+  showPosition, currentLocation, optionLimit, selectedItems, handleChange, handleKeyDownSearch, handleClick, handleKeyDown
+}) => {
+  const data = useStaticQuery(graphql`
+    query {
+			allStrapiSearchCity(
+				sort: {rank: ASC},
+				filter: {rank: {lte: 4}}
+			) {
+				nodes {
+					strapi_id
+					cityName
+					latitude
+					longitude
+					rank
+				}
+			}
+    }
+  `)
+
+  // useState and constants
+  const [searchText, setSearchText] = useState("")
```

Comment:
can we use a different name for searchText e.g. maybe cityText?  It's a bit confusing that is use the same name as the park search field.